### PR TITLE
Create FacesMessagesTest

### LINKED ISSUE #27556: Faces 4.1: FacesMessage: implement equals(), hashcode(), toString()
`https://github.com/jakartaee/faces/issues/1823`

### PR DESCRIPTION
**Pull request by volosied**
RC1 already implemented this change, but I'm adding a test here.

fixes https://github.com/OpenLiberty/open-liberty/issues/27556

### CODE REVIEW

**Comment by volosied** [comment]
#build

File: dev/io.openliberty.org.apache.myfaces.4.1_fat/build.gradle [hunk 1]
```diff
@@ -68,6 +68,7 @@ dependencies {
     requiredLibs 'org.brotli:dec:0.1.2'
     requiredLibs 'xalan:xalan:2.7.2'
     requiredLibs 'xml-apis:xml-apis:1.4.01'
+    requiredLibs project(':io.openliberty.jakarta.faces.4.1')
```
Comment (volosied) [inline dev/io.openliberty.org.apache.myfaces.4.1_fat/build.gradle:71]: Test calls the API directly, so I'm adding the API here.

File: dev/io.openliberty.org.apache.myfaces.4.1_fat/fat/src/io/openliberty/org/apache/myfaces41/fat/test/FacesMessagesTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ * FacesMessage: implement equals(), hashcode(), toString()
+ */
+@RunWith(FATRunner.class)
+public class FacesMessagesTest {
+
+    //These two are the same
+    private static final FacesMessage MESSAGE_1_1 = new FacesMessage(SEVERITY_INFO, "Message", "Details");
+    private static final FacesMessage MESSAGE_1_2 = new FacesMessage(SEVERITY_INFO, "Message", "Details");
+
+    //These differ in each parametergit
```
Comment (pnicolucci) [inline dev/io.openliberty.org.apache.myfaces.4.1_fat/fat/src/io/openliberty/org/apache/myfaces41/fat/test/FacesMessagesTest.java]: `parametergit`

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#libby

**Pull request merged**